core/state/statedb.go

```diff
@@ -159,6 +159,7 @@ func New(root common.Hash, db Database, snaps *snapshot.Tree) (*StateDB, error)
 
 // Quorum
 // NewDual - Create a public and private state from a given public and private tree
+// TODO - rebase - this is not used (except in tests). It was used in the trace APIs. Should we remove it?
```

Comment:
if not used, then we should remove it. should not be left only for tests

Reply:
removed